lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -369,6 +372,7 @@ def response_options(result)
           options[:avs_result] = { code: avs_code_from(result.transaction) }
           options[:cvv_result] = result.transaction.cvv_response_code
         end
+        options[:test] = test?
```

Comment:
we may be able to use `response_options` in `response_from_result`, but I'll do the refactoring in another PR since i'm not sure of the implications of adding `avs_result` and `cvv_result`